dts/arm/nuvoton/m46x.dtsi

```diff
@@ -0,0 +1,586 @@
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		spi9: spi@4006c000 {
+			compatible = "nuvoton,numaker-spi";
+			reg = <0x4006c000 0x6c>;
+			interrupts = <111 0>;
+			reset = <SPI9_RST>;
+			clocks = <&pcc SPI9_MODULE CLK_CLKSEL4_SPI9SEL_HIRC 0>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		spi10: spi@4006d000 {
+			compatible = "nuvoton,numaker-spi";
+			reg = <0x4006d000 0x6c>;
+			interrupts = <119 0>;
+			reset = <SPI10_RST>;
+			clocks = <&pcc SPI10_MODULE CLK_CLKSEL4_SPI10SEL_HIRC 0>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
+		epwm0: epwm@40058000 {
+			compatible = "nuvoton,numaker-pwm";
+			reg = <0x40058000 0x37c>;
+			interrupts = <24 0>, <25 0>, <26 0>, <27 0>;
+			interrupt-names = "brake", "pair0", "pair1", "pair2";
+			reset = <EPWM0_RST>;
+			prescaler = <19>;
+			clocks = <&pcc EPWM0_MODULE CLK_CLKSEL2_EPWM0SEL_PCLK0 0>;
+			#pwm-cells = <3>;
+			status = "disabled";
+		};
+
+		epwm1: epwm@40059000 {
+			compatible = "nuvoton,numaker-pwm";
+			reg = <0x40059000 0x37c>;
+			interrupts = <28 0>, <29 0>, <30 0>, <31 0>;
+			interrupt-names = "brake", "pair0", "pair1", "pair2";
+			reset = <EPWM1_RST>;
+			prescaler = <19>;
+			clocks = <&pcc EPWM1_MODULE CLK_CLKSEL2_EPWM1SEL_PCLK1 0>;
+			#pwm-cells = <3>;
+			status = "disabled";
+		};
+
+		usbd: usbd@400c0000 {
+			compatible = "nuvoton,numaker-usbd";
+			reg = <0x400c0000 0x1000>;
+			interrupts = <53 0>;
+			reset = <USBD_RST>;
+			clocks = <&pcc USBD_MODULE CLK_CLKSEL0_USBSEL_PLL_DIV2 CLK_CLKDIV0_USB(2)>;
+			dma-buffer-size = <1536>;
+			status = "disabled";
+			num-bidir-endpoints = <25>;
+			disallow-iso-in-out-same-number;
+		};
+
+		hsusbd: hsusbd@40019000 {
+			compatible = "nuvoton,numaker-hsusbd";
+			reg = <0x40019000 0x1000>;
+			interrupts = <65 0>;
+			reset = <HSUSBD_RST>;
+			clocks = <&pcc HSUSBD_MODULE 0 0>;
+			dma-buffer-size = <4096>;
+			status = "disabled";
+			num-bidir-endpoints = <(12 + 2)>; /* Including control endpoints by +2 */
+		};
+
+		emac: ethernet@40012000 {
+			compatible = "nuvoton,numaker-ethernet";
+			reg = <0x40012000 0x105C>;
+			interrupts = <66 0>;
+			reset = <EMAC0_RST>;
+			status = "disabled";
+			phy-addr = <1>;
+			clocks = <&pcc EMAC0_MODULE 0 0>;
+		};
+
+		can {
+			compatible = "bosch,m_can-base";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			std-filter-elements = <12>;
+			ext-filter-elements = <10>;
+			rx-fifo0-elements = <3>;
+			rx-fifo1-elements = <3>;
+			rx-buffer-elements = <3>;
+			tx-buffer-elements = <3>;
+
+			canfd0: canfd@40020000 {
+				compatible = "nuvoton,numaker";
+				reg = <0x40020000 0x200>, <0x40020200 0x1800>;
+				reg-names = "m_can", "message_ram";
+				interrupts = <112 0>, <113 0>;
+				interrupt-names = "LINE_0", "LINE_1";
+				reset = <CANFD0_RST>;
+				clocks = <&pcc CANFD0_MODULE CLK_CLKSEL0_CANFD0SEL_HCLK \
+CLK_CLKDIV5_CANFD0(1)>;
```

Comment:
Oh ok, yeah if the CI lets you do it I'd say it's fine, otherwise if you want to indent it that's fine as well but align it with the `&`, something like

````
clocks = <&pcc
          CANFD0_MODULE CLK_CLKSEL0_CANFD0SEL_HCL 
          CLK_CLKDIV5_CANFD0(1)>;
```

Don't think the `\` is needed.